### PR TITLE
Update draft-ietf-dmarc-dmarcbis.md

### PR DESCRIPTION
**Pull request by marmot7**
light editing

### CODE REVIEW

**Comment by marmot7** [comment]
light edits

**Comment by alevesely** [comment]
I took a look at the diffs.  Great work.  However, in some cases the existing text seems to be better.   With reference to old line numbers:

**60** *use of a a domain name*?

**86** *is under no obligation* is more precise than *is not obligated*, IMHO.

**426** *cannot occur with a a message*? (As in **60**).

**705**  *a simple string compare* could be bettered by *a simple string comparison*, not by *a simple string compares*.

**890** *of the Domain Owner or PSO has*, either *of* or *has* must go.

**929** and **986** *The default,* states that this entry is the default and why, not what the default does.

File: draft-ietf-dmarc-dmarcbis.md [hunk 1]
```diff
@@ -53,12 +53,12 @@ This document obsoletes RFC 7489.
...
 
 RFC EDITOR: PLEASE REMOVE THE FOLLOWING PARAGRAPH BEFORE PUBLISHING:
-The source for this draft is maintained in GitHub at:
+The source for this draft is maintained on GitHub at:
 https://github.com/ietf-wg-dmarc/draft-ietf-dmarc-dmarcbis
 
 Abusive email often includes unauthorized and deceptive use of a
-domain name in the "From" header field defined in [@!RFC5322, section 3.6.2] and
-refered to as RFC5322.From. The domain typically
+a domain name in the "From" header field defined in [@!RFC5322, section 3.6.2] and
```
Comment (jrlevine) [inline draft-ietf-dmarc-dmarcbis.md:60]: dup "a a"

File: draft-ietf-dmarc-dmarcbis.md [hunk 2]
```diff
@@ -83,41 +82,40 @@ authenticated for that message. Authentication does not carry an
 explicit or implicit value assertion about that message or about
 the Domain Owner. Furthermore, a mail-receiving organization that performs
 DMARC verification can choose to honor the Domain Owner's requested
-message handling for authentication failures, but it is under no
-obligation to do so; it might choose different actions entirely.
+message handling for authentication failures, but it is not obligated to do so; it might choose different actions entirely.
```
Comment (jrlevine) [inline draft-ietf-dmarc-dmarcbis.md:85]: obligated -> obliged or required

File: draft-ietf-dmarc-dmarcbis.md [hunk 3]
```diff
@@ -83,41 +82,40 @@ authenticated for that message. Authentication does not carry an
...
 reputation.
 
-DMARC policy records can also cover non-existent sub-domains, below the
+DMARC policy records can also cover non-existent sub-domains below the
 "Organizational Domain", as well as domains at the top of the name hierarchy,
 controlled by Public Suffix Operators (PSOs).
 
 DMARC, in the associated [@!I-D.ietf-dmarc-aggregate-reporting] and [@!I-D.ietf-dmarc-failure-reporting] ; 
-documents, also specifies a reporting framework. Using it, a mail-receiving
+documents also specify a reporting framework. Using it, a mail-receiving
```
Comment (toddherr) [inline draft-ietf-dmarc-dmarcbis.md:101]: I don't agree with this change. The subject here is DMARC, not the documents, and therefore the use of "specifies" was correct, i.e., "DMARC also specifies a reporting framework..."

**Comment by toddherr** [comment]
Changes in this pull request were incorporated in the Marmot7 branch - https://github.com/ietf-wg-dmarc/draft-ietf-dmarc-dmarcbis/pull/104

**Pull request closed without merging**